cruise-control-core/src/test/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/RawMetricValuesTest.java

```diff
@@ -206,12 +254,12 @@ public void testAdjacentAvgAtEdgeWhenNewWindowRollsOut() {
     prepareWindowMissingAtIndex(rawValues, NUM_WINDOWS - 1);
 
     assertFalse(rawValues.isValidAtWindowIndex(NUM_WINDOWS - 1));
-    assertFalse(rawValues.hasImputationAtWindowIndex(NUM_WINDOWS - 1));
+    assertFalse(rawValues.isFlawedAtWindowIndex(NUM_WINDOWS - 1));
```

Comment:
It would be more clear to keep the number of terms we use at minimum; hence, maybe we can keep `hasImputationAtWindowIndex` rather than using flawed.